backend/bittan/bittan/api/swish.py

```diff
@@ -0,0 +1,37 @@
+from rest_framework import status 
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework.request import Request
+from ..services.swish import Swish 
+
+@api_view(['POST'])
+def swish_callback(request: Request):
+	# TODO For somereason the swish callback is not working and not sending any data at all. So this is a "temporary" fix.
+	if request.data == {}:
+		# TODO Make this an async job and not execute like this... 
```

Comment:
I can't find an issue for this TODO. To be this sounds important enough to warrant an issue.

Reply:
These are noted in 3. and 4. in #12